.github/workflows/test_workflow_no_dev_install.yml

```diff
@@ -55,6 +55,16 @@ jobs:
           auto-update-conda: true
           python-version: ${{ matrix.PY }}
 
+      - name: Install OpenMDAO Dev
```

Comment:
filename: test_workflow_**no_dev_install**.yml
test name: **Install** OpenMDAO **Dev**

Reply:
yeah, with this PR, we can't run anything without dev.  My choices would be to remove this workflow, or just have it load "dev" until the next release, and then remove that task to go back to a no-dev run.  I went with the latter.

Reply:
I was just pointing out the irony, not requesting changes